invenio_rdm_records/theme/__init__.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+# Copyright (C) 2020 Northwestern University.
+#
+# Invenio RDM Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio RDM Records."""
+
+from ..version import __version__
+
+__all__ = ('__version__', )
```

Comment:
why is this file necessary?

Reply:
I think is not. It's a port from cookiecutter-invenio-instance/records module. I think the file is, the import of version is not: https://github.com/inveniosoftware/invenio-rdm-records/issues/63

Reply:
> the import of version is not

Haha that's what I meant. I am fine with the presence of __init__.py :)